apps/backoffice-v2/src/lib/blocks/hooks/useCaseInfoBlock/useCaseInfoBlock.tsx

```diff
@@ -15,7 +15,12 @@ export const useCaseInfoBlock = ({
   entityDataAdditionalInfo: TWorkflowById['context']['entity']['data']['additionalInfo'];
 }) => {
   return useMemo(() => {
-    if (Object.keys(entity?.data ?? {}).length === 0) {
+    const entityDetails = [
+      ...Object.entries(omitPropsFromObject(entity?.data, 'additionalInfo', 'address') ?? {}),
+      ...Object.entries(omitPropsFromObject(entityDataAdditionalInfo ?? {}, 'ubos')),
+    ];
+
+    if (Object.keys(entityDetails ?? {}).length === 0) {
```

Comment:
Fixed empty entity info block state. Until now we checked length on unused keys.